config/locales/devise.en.yml

```diff
@@ -47,7 +47,7 @@ en:
       updated_but_not_signed_in: "Your account has been updated successfully, but since your password was changed, you need to sign in again"
     sessions:
       signed_in: "Signed in successfully."
-      signed_out: "Signed out successfully."
+      signed_out: You have logged out of the Competitions Service Provider. However, you are still logged into UNA and any other SAML authenticated applications. Please visit the <a href="https://auth.nih.gov/siteminderagent/smlogout.asp?mode=nih&AppReturnUrl=https://competitions-cd2h-staging.fsm.northwestern.edu">UNA logout</a> and close your browser to end this session. Visit <a href="https://uit.stanford.edu/service/saml/logout">SAML Logout</a> for more details on this process.
```

Comment:
Suggested edit: 
You have logged out of Competitions but are still logged into UNA and any other Shibboleth authenticated applications. Please quit your browser or [link]logout of UNA[link]to end this session.

Reply:
Can we hardcode the UNA log out link into the navigation partial to log users out?

Reply:
For the second part it seems like that should be possible, but I think it would take overriding some of the devise defaults. I also believe that once UNA has the single signout - that is what id does, but I don't think we can initiate a single sign out from our application currently. I changed  the text in the first edit.